src/backend/sqlite/index.rs

```diff
@@ -6,6 +6,7 @@ impl IndexBuilder for SqliteQueryBuilder {
             write!(sql, "CONSTRAINT ").unwrap();
         }
         self.prepare_index_name(&create.index.name, sql);
+        write!(sql, " ").unwrap();
```

Comment:
Hey @nahuakang, once again, thanks for contributing!!

Logically, the space should be added only when the constraint is named. The code should be...

```suggestion
            write!(sql, "CONSTRAINT ").unwrap();
            self.prepare_index_name(&create.index.name, sql);
            write!(sql, " ").unwrap();
        }
```

However, the existing code works just fine because there is a checking where space cannot be appended to the SQL string where it already end with a space
- https://github.com/SeaQL/sea-query/blob/190eb5041e44173a560bb252f1dd4dd6d07fc77b/src/prepare.rs#L82-L94

Reply:
Thanks for this code snippet! Good to know :D